infra/main.bicep

```diff
@@ -1,19 +1,12 @@
 targetScope = 'subscription'
 
 @minLength(1)
-@maxLength(18)
-@description('Name of the environment that can be used as part of naming resource convention')
+@maxLength(64)
+@description('Name of the the environment which is used to generate a short unique hash used in all resources.')
 param environmentName string
 
 @minLength(1)
 @description('Primary location for all resources')
-@allowed([
-  'australiaeast'
-  'centralindia'
-  'eastus'
-  'uksouth'
-  'westeurope'
-])
```

Comment:
I intentionally added this location restriction because APIC is currently served within these regions. Can we put this back?

Reply:
Let's use the azd location type: https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/infra/main.bicep#L57

That way they get the normal prompt but filtered to the locations that work for the template.